tools/file_packager.py

```diff
@@ -604,59 +604,117 @@ def was_seen(name):
         };
       };
 
+      // Gives us room for 100 char key names
+      var CHUNK_SIZE = 133169152 - 100;
+
+      function cacheRemotePackage(
+        db,
+        packageName,
+        packageData,
+        packageMeta,
+        callback,
+        errback
+      ) {
+        var transaction_packages = db.transaction([PACKAGE_STORE_NAME], IDB_RW);
+        var packages = transaction_packages.objectStore(PACKAGE_STORE_NAME);
+        var chunkSliceStart = 0;
+        var chunkCount = Math.floor(packageData.byteLength / CHUNK_SIZE) + 1;
+        var finishedChunks = 0;
+        for (var chunkId = 0; chunkId < chunkCount; chunkId++) {
+          Module.print('Iter...', chunkId, chunkCount, chunkSliceStart, packageName);
+          var putPackageRequest = packages.put(
+            packageData.slice(chunkSliceStart, (chunkSliceStart += CHUNK_SIZE)),
```

Comment:
That was just in the interest of efficiency (not needing to set another variable), how should I go about this then? I don't want to use multiplication since we can already `+=` it and it'd be wasteful

Reply:
@kripken The issue with that is that then we can't use the existing value for the slice start

Reply:
True, yeah, we'd need a temp variable. How about something like this?
```
var nextChunkSliceStart = chunkSliceStart + CHUNK_SIZE;
var putPackageRequest = packages.put(
  packageData.slice(chunkSliceStart, nextChunkSliceStart),
  'package/' + packageName + '/' + chunkId
);
chunkSliceStart = nextChunkSliceStart;
```